### ISSUE TITLE
Init requires a a token not API key

### ISSUE THREAD

**Issue by tomgoldsmith** [mixpanel/mixpanel-node#97]
Hello a couple of suggestions to make it a bit easier to understand the documentation. 

Would it be possible to update the documentation to explicitly say that the init function requires the token?

In addition it would be helpful if it explained that "billybob" could be substituted for the distinct id of the user. Fairly obvious when you get into it bit just something that you should not have to check. 

Happy to add this via pull request if you would like. 

Thanks!

**Comment by cspeer** [comment]
Good call! I also tripped over this. I naturally assumed the secret would be needed here.

**Comment by tdumitrescu** [comment]
Sure, there are a lot of things we tend to take for granted in the Mixpanel libs that could be clearer, such as that you can't use our tracking libs without a token. If you'd like to throw together a quick PR with changes you'd like in the docs, that would be super helpful!
